frontend/src/entry/overview/EntryRoleList.tsx

```diff
@@ -0,0 +1,23 @@
+import { Grid } from '@mui/material'
+import EntryCardEntityRoleRow from 'src/entry/overview/EntryCardEntityRoleRow'
+import { EntryInterface } from 'types/types'
+
+type EntryCardRoleDialogProps = {
+  entry: EntryInterface
+}
+
+export default function EntryRoleList({ entry }: EntryCardRoleDialogProps) {
+  return (
+    <>
+      <Grid container spacing={2} paddingX={5} paddingBottom={2}>
```

Comment:
`paddingX` and `paddingBottom` can be removed. This padding will be handled by using `<DialogContent>` as suggested in another comment.